_sass/head.scss

```diff
@@ -0,0 +1,8 @@
+.nav-list .nav-list-item .nav-list-link {
+  background-image: linear-gradient(
+    -90deg,
+    rgba($feedback-color, 1) 0%,
+    rgba($feedback-color, 0.8) 80%,
+    rgba($feedback-color, 0) 100%
+  );
+}
```

Comment:
None of the other files in `assets/css` include style rules, so I thought that it would be a bit easier to find these when they are in `_sass` (and most of the other files there are imported only once!). But that isn't a strong argument, and I'm happy to inline it.